int32[] pid_gains_for_posture
int32[] pid_gains_for_linear_velocity
int32[] pid_gains_for_angular_velocity
int32[] pid_gains_for_position_control
